web/src/emoji_picker.js

```diff
@@ -619,71 +619,65 @@ function register_popover_events($popover) {
     });
 }
 
-export function build_emoji_popover(element, id) {
-    const $elt = $(element);
-    if (id !== undefined) {
-        message_lists.current.select_id(id);
-    }
+function get_default_emoji_popover_options() {
+    const template = render_emoji_popover();
 
-    let placement = popovers.compute_placement($elt, APPROX_HEIGHT, APPROX_WIDTH, true);
+    return {
+        placement: "top",
+        onCreate(instance) {
+            if (current_message_id) {
+                message_lists.current.select_id(current_message_id);
+            }
+            emoji_popover_instance = instance;
+            const $popover = $(instance.popper);
+            $popover.addClass("emoji-popover-root");
+            instance.setContent(ui_util.parse_html(template));
+            $popover
+                .find(".popover-content")
+                .append(generate_emoji_picker_content(current_message_id));
+            emoji_catalog_last_coordinates = {
+                section: 0,
+                index: 0,
+            };
+        },
+        onShow(instance) {
+            const $reference = $(instance.reference);
+            $reference.addClass("reaction_button_visible");
+            $reference.parent().addClass("reaction_button_visible");
+        },
+        onMount(instance) {
+            const $popover = $(instance.popper);
+            refill_section_head_offsets($popover);
+            register_popover_events($popover);
+            show_emoji_catalog();
+        },
+        onShown() {
+            change_focus_to_filter();
```

Comment:
This doesn't seem to be working after you try to open emoji popover for the 2nd time.

Reply:
Fixed, thanks!